app/uk/gov/hmrc/cipphonenumberverification/services/VerifyService.scala

```diff
@@ -37,9 +41,13 @@ class VerifyService @Inject()(passcodeService: PasscodeService,
     } yield result
 
   def verifyOtp(phoneNumberAndOtp: PhoneNumberAndOtp)(implicit hc: HeaderCarrier): Future[Result] = {
-    for {
+    (for {
       httpResponse <- validatorConnector.callService(phoneNumberAndOtp.phoneNumber)
       result <- processResponseForOtp(httpResponse, phoneNumberAndOtp)
-    } yield result
+    } yield result).recover {
```

Comment:
please remove recover block, I removed it as part of CAV-324

Reply:
Thank you that was a rebase issue, i thought i had to keep my version.

Reply:
I have put it back